script.js

```diff
@@ -0,0 +1,29 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const allShapes = ['rock','paper','scissors'];
```

Comment:
please use either double quotes (`"abc"`) or single quotes (`'abc'`), but not a mix of both

Reply:
also, if constants are declared like that, it is better to be using all upper case names (`const ROCK`, `const ALL_SHAPES`)